src/utils/ajax.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
We're using redux to fetch data. you can check out some of the redux actions i have done, like login, register. If it's confusing, don't worry. I'd work on this.